Track.h

```diff
@@ -109,12 +127,20 @@ class Track
 private:
   TrackState state_;
   HitVec hits_;
-  HitVec initHits_;
-  HitIdxVec hitIdxVec_;
   float chi2_;
+  unsigned int mcTrackID_;
+  unsigned int nHitsMatched_;
+  unsigned int seedID_;
+  unsigned int duplicateID_;
+  bool isDuplicate_;
+  HitIdxVec hitIdxVec_;
```

Comment:
Hi @kmcdermo, are you suggesting to merge this one as it is and then go through ttree-merge-2-int?